pontoon/base/forms.py

```diff
@@ -65,16 +65,13 @@ def clean(self):
 
             # File format validation
             if part:
-                file_extension = os.path.splitext(uploadfile.name)[1].lower()
-                part_extension = os.path.splitext(part)[1].lower()
-
-                # For now, skip if uploading file while using subpages
-                if (
-                    part_extension in SUPPORTED_FORMAT_PARSERS.keys()
-                    and part_extension != file_extension
-                ):
+                uploadfile_ext = os.path.splitext(uploadfile.name)[1].lower()
```

Comment:
[nitpick/advice] In Python 3, you can use [Pathlib](https://docs.python.org/3/library/pathlib.html) objects.
```
Path(uploadfile.name).suffix
```